.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
             "program": "${workspaceRoot}/src/azure-cli/azure/cli/__main__.py",
             "cwd": "${workspaceRoot}",
             "args": [
-                "--help"
+                "iot", "hub", "list", "--debug"
```

Comment:
We usually don't track `.vscode/launch.json`.

Reply:
Changed.